server/translations/de.json

```diff
@@ -18,6 +18,7 @@
 		"rankings": "Platzierungen",
 		"meets": "Wettkämpfe",
 		"data": "Daten",
+                "status": "Project Status",
```

Comment:
These lines have some different spacing (in each of the JSON files).

Reply:
Odd, not seeing the different spacing in vim